docs/src/man/hosting.md

```diff
@@ -340,6 +339,21 @@ or `git@`.
 See the [`deploydocs`](@ref) function documentation for more details.
 
 
+## Using Simple SSH
+
+If you do not have access to either Travis or Github for automated documentation updates,
+or you are working in a private git hosting service where using these is impractical, it
+may be desirable to push objects more simply via SSH.  For such cases, simply call, e.g.
+
+```julia
+deploydocs(
+    repo = "github.com/USER_NAME/PACKAGE_NAME.jl.git",
+    deploy_config = Documenter.SimpleSSHConfig("/path/to/ssh-key")
+)
```

Comment:
Should we maybe recommend guarding the `deploydocs` call here, e.g. with something like `"deploy" in ARGS && deploydocs(...)`? It would always deploy deploy when you're building docs locally at the moment?

Reply:
Sounds good, commit incoming.